docs/reports/phase3-summary.md

```
@@ -242,9 +242,9 @@ sudo tail /var/log/pgvector_monitor.log
 ### Manual Steps (if needed)
 ```bash
 # Copy files to runtime
-sudo cp /home/tommy/.dev/ai-orchestrator/deploy-phase3.sh /opt/ai-orchestrator/
-sudo cp /home/tommy/.dev/ai-orchestrator/docker-compose.yml /opt/ai-orchestrator/
-sudo cp -r /home/tommy/.dev/ai-orchestrator/tools /opt/ai-orchestrator/
+sudo cp ./deploy-phase3.sh /opt/ai-orchestrator/
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)